Fix eslint errors

The following issues have been found after linting via eslint and should be fixed:
```
index.js
   29:20  warning  Unexpected 'else' after 'return'  no-else-return
  104:14  error    Unnecessary semicolon             no-extra-semi
  164:20  warning  Unexpected 'else' after 'return'  no-else-return

tests\index.js
    1:1   error    Use the global form of "use strict"  strict
  120:56  warning  Strings must use singlequote         quotes
  222:47  warning  Strings must use singlequote         quotes
```

--- a/index.js
+++ b/index.js
@@ -26,15 +26,15 @@ function icylog(opts) {
 
             if (err) {
                 return next(unexpectedHandler(err));
-            } else {
-                error = results[0];
-                handler = results[1];
-                payload = results[2];
-
-                handler(error, function () {
-                    handleHttpResponse(error, req, res, payload);
-                });
             }
+			
+            error = results[0];
+            handler = results[1];
+            payload = results[2];
+
+            handler(error, function () {
+                handleHttpResponse(error, req, res, payload);
+            });
         });
 
         function handleHttpResponse(err, req, res, payload) {
@@ -101,7 +101,7 @@ function icylog(opts) {
                     debug('Failed to elect handler');
                     return callback(new Error('The HTTP Status Code is neither 4xx or 5xx. Refusing to treat as Error'));
                 }
-            };
+            }
 
             function clientHandler(err, callback) {
                 debug('Invoking Client handler');
@@ -161,15 +161,15 @@ function icylog(opts) {
 
             if (err) {
                 return callback(unexpectedHandler(err));
-            } else {
-                error = results[0];
-                handler = results[1];
-                payload = results[2];
-
-                handler(error, function () {
-                    handleWsResponse(payload, callback);
-                });
             }
+			
+            error = results[0];
+            handler = results[1];
+            payload = results[2];
+
+            handler(error, function () {
+                handleWsResponse(payload, callback);
+            });
         });
 
         function handleWsResponse(payload, callback) {

--- a/tests/index.js
+++ b/tests/index.js
@@ -1,4 +1,6 @@
-﻿var test = require('tape'),
+﻿'use strict';
+
+var test = require('tape'),
     express = require('express'),
     request = require('supertest'),
     icylog = require('../index');
@@ -117,7 +119,7 @@ test('Should refuse to invoke httpHandler', function (t) {
         .expect(500)
         .end(function (err, res) {
             t.error(err, 'should not exist');
-            t.equal(res.body.message.substring(0, 23), "An unexpected exception", 'should match');
+            t.equal(res.body.message.substring(0, 23), 'An unexpected exception', 'should match');
             t.end();
         });
 });
@@ -219,7 +221,7 @@ test('Should refuse to invoke wsHandler', function (t) {
     error.status = 200;
 
     errorHandler(error, function (err, payload) {
-        t.equal(err.message.substring(0, 23), "An unexpected exception", 'should match');
+        t.equal(err.message.substring(0, 23), 'An unexpected exception', 'should match');
         t.end();
     });
 });

